src/server/_params.py

```diff
@@ -381,7 +381,10 @@ def _parse_range(part: str):
 def parse_date(s: str) -> int:
     # parses a given string in format YYYYMMDD or YYYY-MM-DD to a number in the form YYYYMMDD
     try:
-        return int(s.replace("-", ""))
+        if s == "*":
+            return s
+        else:
+            return int(s.replace("-", ""))
```

Comment:
```suggestion
    if s == "*":
        return s
    try:
        return int(s.replace("-", ""))
```
the `if "*"` doesnt need to be in the `try` block (which is there to make sure `s` is made up of numbers (and dashes)), and the `else` is superfluous since the `return` happens right above it.